### PR TITLE
Add python client example in user guide

### PR DESCRIPTION
**Pull request by jt2594838**
Refine the python client example readme and also put it into the user guide.

### CODE REVIEW

**Comment by jt2594838** [comment]
> considering #445 ?

I suggest keeping these scripts as you can go without maven.

File: docs/Documentation-CHN/OtherMaterial-Examples.md [hunk 1]
```diff
@@ -81,3 +81,172 @@ public class IotdbHelloWorld {
...
+
+if __name__ == '__main__':
+    ip = "localhost"
+    port = "6667"
+    username = 'root'
+    password = 'root'
+    # Make socket
+    transport = TSocket.TSocket(ip, port)
+
+    # Buffering is critical. Raw sockets are very slow
```
[the remark this answers is not in the document]
Comment (jt2594838) [reply docs/Documentation-CHN/OtherMaterial-Examples.md]: Other example codes also use English comments and I do not think this is a big issue.

**Comment by jixuan1989** [comment]
1. I'd like to know which python version it needs... python2 or 3?
2. how to use it...

```
MacBook-Pro-2:src hxd$ python3 client_example.py
Traceback (most recent call last):
  File "client_example.py", line 22, in <module>
    from thrift.protocol import TBinaryProtocol
ModuleNotFoundError: No module named 'thrift'
```
pip install thrift?

**Comment by jixuan1989** [comment]
and, please add execute privilege to the sh and py files.

**Comment by jixuan1989** [comment]
you may need to add some content in readme.md like:

```
before running the example, please make sure the thrift module is installed. 
e.g.,
pip install thrift==0.9.3
```

However, even I install the thrift, I can not run the example successfully.
```
MacBook-Pro-2:src hxd$ python client_example.py
Traceback (most recent call last):
  File "client_example.py", line 25, in <module>
    from rpc.TSIService import Client, TSCreateTimeseriesReq, TSInsertionReq, \
  File "../target/rpc/TSIService.py", line 9, in <module>
    from thrift.Thrift import TType, TMessageType, TFrozenDict, TException, TApplicationException
```

**Comment by jt2594838** [comment]
I think I have answered all your questions. I do not know why you have skipped them.
About the python version and how to install thrift.
image

About the missing library.
image

**Pull request merged**